### DISCUSSION TITLE
Transform tool, Guides

### DISCUSSION THREAD

**Discussion by alexvoz** [PintaProject/Pinta#732] [Ideas]
Add classic transform tool (nine-point border) to move, scale and rotate objects/layers on canvas
![image](https://github.com/PintaProject/Pinta/assets/6594335/820174c8-fc96-4b25-9471-70939bc82b2c)
Add guides on canvas. Snap objects to guides

**Comment by cameronwhite** [comment]
This is mostly covered by #585 which has some in-progress work

**Comment by bakhsv** [comment]
Guides would be extremely helpful. That's the only feature I'm missing in Pinta.
